hummingbot/connector/exchange/bittrex/bittrex_exchange.pyx

```diff
@@ -347,7 +347,19 @@ cdef class BittrexExchange(ExchangeBase):
             open_orders = dict((entry["id"], entry) for entry in open_orders)
 
             for tracked_order in tracked_orders:
-                exchange_order_id = await tracked_order.get_exchange_order_id()
+                try:
+                    exchange_order_id = await tracked_order.get_exchange_order_id()
+                except asyncio.TimeoutError:
+                    if tracked_order.last_state == "FAILURE":
+                        self.c_stop_tracking_order(client_order_id)
+                        self.logger().warning(
+                            f"No exchange ID found for {client_order_id} on order status update."
+                            f" Order no longer tracked. This is most likely due to a POST_ONLY_NOT_MET error."
+                        )
+                        continue
```

Comment:
I don't think we need the `continue` here, because there is no more logic after this point in the for cycle

Reply:
Lines 363 to 431 (end of the function) are also under the loop's cycle